waspc/examples/todoApp/ext/queries.js

```diff
@@ -0,0 +1,4 @@
+
+export const getTasks = async ({ args, context }) => {
+  return [] // TODO: This is dummy for now.
```

Comment:
aha so here would wasp developer write query to db, right?

Reply:
Yes, exactly.